stl/inc/random

```diff
@@ -2025,6 +2027,39 @@ basic_ostream<_Elem, _Traits>& operator<<(basic_ostream<_Elem, _Traits>& _Ostr,
     return _Dist._Write(_Ostr);
 }
 
+// FUNCTION TEMPLATE _Float_upper_bound
+
+// Returns smallest _Flt s.t. static_cast<_Ty>(_Result) > _Val. First
+// truncate to largest _Flt <= _Val, then add ceil(ulp)
+
+template <class _Flt, class _Ty, typename = enable_if_t<is_unsigned_v<_Ty> && is_floating_point_v<_Flt>>>
+_NODISCARD inline _Flt _Float_upper_bound(_Ty _Val) {

```

Comment:
A year or two ago, we changed our conventions - now we avoid saying `inline` for `template` functions (which are already header-only (if they aren't explicit specializations)). `inline` had very little influence on optimizer inlining decisions and we largely trust it to do the right thing now.
```suggestion
_NODISCARD _Flt _Float_upper_bound(_Ty _Val) {
```